### ISSUE TITLE
Please allow disabling of call confirmation prompt

### ISSUE THREAD

**Issue by 1xPdd** [vector-im/riot-android#2778]
I know the UI revision is on the horizon, but I'd like to make a plug for simplifying the number of clicks/taps from open to making call. By my count, it's four. I tried setting up Riot on some family and friend's phones and this redundancy adds to the sense of confusion that non-tech people have with the UI. Please consider adding an option to disable to final confirmation and (possibly) to change the call button + prompt for type (voice/video) into two separate voice & call buttons.

**Comment by bmarty** [comment]
I agree that the call confirmation dialog is not necessary when you have chosen between voice call or video call because you already had the possibility to cancel the action.
